### PR TITLE
[Quick Fix] update ctime for utimes syscalls

### PR DESCRIPTION
**Pull request by grief8**
According to the [linux manual](https://man7.org/linux/man-pages/man2/utime.2.html), the utime() system call changes the status change time (ctime) to the current time, even if the other time stamps don't actually change.
> The utime() system call changes the access and modification times
       of the inode specified by filename to the actime and modtime
       fields of times respectively.  The status change time (ctime)
       will be set to the current time, even if the other time stamps
       don't actually change.

### CODE REVIEW

File: kernel/aster-nix/src/syscall/utimens.rs [hunk 1]
```diff
@@ -140,6 +140,8 @@ fn vfs_utimes(dentry: &Arc<Dentry>, times: Option<TimeSpecPair>) -> Result<Sysca
     // Update times
     dentry.set_atime(atime);
     dentry.set_mtime(mtime);
+    let now = RealTimeCoarseClock::get().read_time();
```
Comment (liqinggd) [inline kernel/aster-nix/src/syscall/utimens.rs:143]: can we return the `ctime` in line 112 ? so there is no need to call `RealTimeCoarseClock::get().read_time()` multiple times.

Comment (grief8) [reply kernel/aster-nix/src/syscall/utimens.rs]: That's better! Thank you for your reminder.

**Comment by liqinggd** [comment]
LGTM

**Review by tatetian — APPROVED** [review]
LGTM. Thanks for the contribution!

**Pull request merged**